gensrc/thrift/FrontendService.thrift

```diff
@@ -1511,6 +1511,29 @@ struct TUpdateResourceUsageResponse {
     1: optional Status.TStatus status
 }
 
+struct TGetQueryStatisticsRequest {
+}
+
+struct TQueryStatisticsInfo {
+    1: optional i64 queryStartTime
+    2: optional string queryId
+    3: optional string connId
+    4: optional string db
+    5: optional string user
+    6: optional i64 cpuCostNs
+    7: optional i64 scanBytes
+    8: optional i64 scanRows
+    9: optional i64 memUsageBytes
+    10: optional i64 spillBytes
+    11: optional i64 execTime
+    12: optional string wareHouseName
```

Comment:
for global_current_queries, better to add a field to describe which FE the query comes from

Reply:
Add a new field feIp. See latest PR